control/plugin/plugin.go

```diff
@@ -91,7 +91,7 @@ var (
 		"collector",
 		"processor",
 		"publisher",
-		"streamCollector",
+		"streaming collector",
```

Comment:
I suggest changing type name to "streaming-collector" instead of having space there. The reason is to avoid having the skip space for example in command `snaptel plugin unload <plugin_type> <plugin_name> <plugin_version>`